### DISCUSSION TITLE
Can I create a user from a form in a public screen?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#3373] [Help]
Hello,

I hope.you are fine.

I would like to know if it were possible to add a user (Id +password) using a form in a public view?

Or maybe by automation?

Thanks a lot

**Comment by SantoshSrinivas79** [comment]
Hi @oswaldB,

I believe user creation from a public form isn't going to be supported.
However, [this](https://github.com/Budibase/budibase/discussions/1975#discussioncomment-1475898) could be a way to create a hack for it.

I haven't yet tried it. I am still facing major issues trying to understand the RBAC methodology before I try it out.

Cheers!

**Comment by shogunpurple** [comment]
Hi @oswaldB - we do not support this and probably never will for the reasons detailed here: https://github.com/Budibase/budibase/discussions/1975#discussioncomment-997807

Budibase aims to be deliberate with how it manages users with a more organisation centred approach - controlled through RBAC with other internal users invited directly by admin users.

If you just want to collect email addresses or something similar, you can of course create another internal or external table and expose a public form to add new emails/user information, but will not be budibase users that can sign into the platform. Would this be sufficient?
